helevents/service_tasks/hourly_tasks.sh

```diff
@@ -11,9 +11,14 @@ fi
 
 echo "--- Starting kulke importer ---"
 
+timeout --preserve-status -s INT 30m python manage.py event_import kulke --keywords
```

Comment:
Not strictly related to this PR but the total max timeouts in the hourly cron are now 10 + 30 + 30 + 45 + 75 plus a bunch of commands without timeouts. That's like 190 minutes. 

In any case, Kubernetes/Openshift has `activeDeadlineSeconds` which allows setting a max time for the whole (cron)job. Currently not set. Could consider.

Reply:
🤔 good point. Should probably do something about this.